server/db/create_test_data.js

```diff
@@ -44,7 +45,7 @@ function createTestUser(recordCount, cb) {
     var visitedAt = new Date(historyDate.getTime() + (1000 * n)).toJSON();
     var title = 'Title for Page Number ' + n;
     var url = 'https://www.record' + n + '.com/whatever';
-    var urlHash = crypto.createHash('sha1').update(url).digest('hex').toString();
+    var urlHash = utils.createUrlHash(url);
```

Comment:
Refactored this out into `utils.js` since it seemed we were using `crypto.createHash('sha1').update(url).digest('hex').toString();` in at least 3 places.